Tests/Tests/PromiseKit/PromiseTests.swift

```diff
@@ -1006,6 +1006,78 @@ extension PromiseTests {
   }
 }
 
+// MARK: - Sequence tests
+
+extension PromiseTests {
+  func testSequence1() {
+    func next(_ idx: Int) -> Promise<Int>? {
+      guard idx < 10 else { return nil }
+      return async(idx, delay: .random(in: 0.01...0.5))
```

Comment:
Any particular reason why this is 10? Also, that in combination with the potentially long random delay 0.5, causes this test to go longer than 2 seconds.

Reply:
Altered the interval to `0.01...0.05` so now the test runs for at most 0.5s.